### ISSUE TITLE
How to upgrade from v2.3.0 to GitHub release

### ISSUE THREAD

**Issue by gailhoskins** [phw198/OutlookGoogleCalendarSync#423]
**OGCS Version**: v2.3.0
**Installed or Portable**: Installed

**Problem Description**  
I have been using OGCS for over a year and version 2.3.0 for probably ten months or more. All was good through 01/05/18. As of 1/6/18 I get the message “Problem encountered during synchronisation.
Object reference not set to an instance of an object.” I followed the info and got over to you on github and I figure that I should upgrade to 2.6.0 but *I do not know* if I should uninstall the old version first or just download and install the new one. Also my info on Outlook is super important to me. I don’t mind losing what is on google calendar. Please tell me *how I am* I supposed to indicate which is the “primary” source so nothing bad happens. Thanks for you help. Gail H in Ohio

**Comment by phw198** [comment]
v2.3.0 hey? Does that mean you haven't been getting prompted to upgrade each time you started OGCS?

Anyway, the migration to GitHub was carefully handled by each subsequent release of the application, where it would automatically get rid of CodePlex's installation in favour of GitHub's installer - migrating your settings etc.

So I'd suggest you:
1. [Upgrade to v2.5.0](http://outlookgooglecalendarsync.codeplex.com/downloads/get/clickOnce/OutlookGoogleCalendarSync.application) which is the last CodePlex release
1. Perform a sync. Even if this release has fixed the problem, you should still continue upgrading.
1. Go to the `About` tab and click `Check for Update`
1. Upgrade to v2.6.0
1. If the problem still exists, check `Check for Alpha releases` and upgrade again.

To keep your Outlook data safe you can do the following - remember to click `Save` to make the settings persist through restarts of OGCS:
* Set it to one-way sync from Outlook to Google (assuming you currently have it two-way syncing)
* Enable `Disable deletions` or `Confirm deletions` on the `Settings > Sync Options` tab.

Hope that helps!

**Comment by gailhoskins** [comment]
Thank you very much for both the prompt reply and the tech-level advice that fit my needs. 

I may have gotten one or two notices perhaps six or eight months ago about 2.3.0 needing to be updated. I believe I visited the old site and then saw it was soon to be migrated to github and I figured I would wait for things to get settled. I certainly have not gotten any alerts to update in months. I followed your advice and got to 2.5 and then to 2.6.0 and all is good there. I am amazed and impressed and grateful for the responsiveness from you. And delighted I did not have to start over setting up my settings.

Gail
